genai-perf/genai_perf/telemetry_data/triton_telemetry_data_collector.py

```diff
@@ -66,34 +66,56 @@ def _process_and_update_metrics(self, metrics_data: str) -> None:
             ```
 
             The method will extract and process:
-            - `nv_gpu_power_usage` as `gpu_power_usage`
-            - `nv_gpu_utilization` as `gpu_utilization`
-            - `nv_energy_consumption` as `energy_consumption`
+            'gpu_power_usage': {
+                'gpu0': [27.01]
```

Comment:
This is sample data?

Reply:
That's the sample how the method stores the data.